src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/PackageDeclarationCheckTest.java

```diff
@@ -69,6 +70,29 @@ public void testOnFileWithCommentOnly() throws Exception {
     @Test
     public void testCorrectFile() throws Exception {
```

Comment:
test name shoudl explain its purpose. You are testing ```ignoreMismatch``` options which is set to true.